<launch>
  <!-- Use a different machine name to connect to a different ROS server-->
    <machine name="local" address="localhost" default="true"/>
    <arg name="input_topic" default="/camera/depth_registered/points" />
    <arg name="output_file" default="/net/home/janani.mohan/test_ws/src/cloud2.pcd" />
    <node pkg="pcd_writer" type="pcd_writer_node" name="pcd_writer" output="screen" respawn="false">
        <param name="input_topic" value="$(arg input_topic)" />
        <param name="output_file" value="$(arg output_file)" />
    </node>
</launch>
